.env.example

```diff
@@ -1,6 +1,7 @@
 ALLOWED_HOSTS=*
 DEBUG=True
+MODE=DEVELOPMENT
 SECRET_KEY=2gr6ud88x=(p855_5nbj_+7^bw-iz&n7ldqv%94mjaecl+b9=4
 DJANGO_DATABASE_KEY=default
 DJANGO_DATABASE_URL=postgres://mathesar:mathesar@db:5432/mathesar_django
-MATHESAR_DATABASES=(mathesar_tables|postgresql://mathesar:mathesar@db:5432/mathesar)
+MATHESAR_DATABASES=(mathesar_tables|postgresql://mathesar:mathesar@db:5432/mathesar)
```

Comment:
Please add the newline back to the end of the file.

Reply:
It seems like dotenv is reading the newline and throwing a warning saying that the line has invalid env variable configuration. I removed it to avoid the warning. I will add it back for consistency.

Reply:
Added the newline back.